tests/lib/ocpp/v201/test_charge_point.cpp

```diff
@@ -158,32 +123,6 @@ class ChargePointFixture : public DatabaseTestingUtils {
                                .transactionId = transaction_id};
     }
 
-    ocpp::v201::Callbacks create_callbacks_with_mocks() {
-        ocpp::v201::Callbacks callbacks;
-
-        callbacks.is_reset_allowed_callback = is_reset_allowed_callback_mock.AsStdFunction();
```

Comment:
please clarify why these were removed? I don't see where they were replaced (at least while skimming through the changes), and I don't see any explanation in the PR about why they are no longer needed.

Reply:
create_callbacks_with_mocks() created a Callbacks object, then setup the mocks on it, then returned it.

But we already have a Callbacks object in the test fixture, with a function that is used to setup the mocks on it.

These functions were almost identical, except one used the fixture's Callbacks object and one created a new one. So I eliminated the need for one of them by using the other.